www/xyerv00/DU3_index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="DU3_style.css">
+</head>
+
+<body>
+<div>The font of this web page is not serif</div>
+<div class="p-red">This text is red-ish using a named color</div>
+<div class="p-green">This text is green-ish using a rgb color</div>
+<div class="p-blue-bold">This text is blue-ish using a hex color and the text is bold</div>
+<div class="p-size">The size of this text is 18px</div>
+<div class="p-italic">The size of this text is 24px and the text is italic</div>
+<div class="p-white">This text is white and the background color is brown-ish</div>
+<p class="p-centered">This paragraph's content is centered and the line height is 32px. Lorem ipsum dolor sit
+    amet consectetur
+    adipisicing elit.
+    Earum excepturi optio consectetur molestias deleniti, iure voluptas reiciendis, dolor iste voluptatem iusto
+    aliquid unde, quam commodi a doloribus impedit numquam assumenda.</p>
+<p class="p-justified">This paragraph's content is justified and the text is capitalized everywhere (use
+    text-transform). Lorem ipsum
+    dolor, sit amet consectetur adipisicing elit. Expedita ipsa perferendis quod quia obcaecati illum enim saepe,
+    maxime, ratione iure iusto. Quae delectus voluptatibus, repellendus veniam facere molestias dolor possimus!</p>
+<div class="boxOne">This box is 400px in width and 300px in height total. It has a solid 1px purple border and
```

Comment:
používej jen jeden styl zápisu názvu tříd a id, doporučuju to psát s pomlčkami, např. box-one, box-two...

Reply:
OK, mela jsem trochu problem s ideou, protoze mi linkovala s classy v jinych souborech, ktere jsou v projektu, proto jsem chtela mit sve nazvy. Takze to mam predelat?